requirements.txt

```diff
@@ -1,4 +1,3 @@
-coverage==4.1
```

Comment:
Felt like maybe this didn't need to be in the requirements that get imported into setup.py? Test settings got separated anyway to remove the django requirement so tox could take care of that (gives duplicate error if in both).